tortoise/verifying_tortoise.go

```diff
@@ -566,42 +571,43 @@ func (t *turtle) processBlock(ctx context.Context, block *types.Block) error {
 			opinion[blk] = nvote
 		}
 	}
-	t.BlockLayer[block.ID()] = block.LayerIndex
-	logger.With().Debug("adding or updating block opinion")
-	t.BlockOpinionsByLayer[block.LayerIndex][block.ID()] = opinion
+	t.BallotLayer[ballot.ID()] = ballot.Layer()
```

Comment:
it should be LayerIndex, there is also a Layer object that stores collection of blocks

Reply:
done